### PR TITLE
Add React List of goods

### PR DESCRIPTION
**Pull request by Ruslan-Pashkov**
[DEMO LINK](https://ruslan-pashkov.github.io/react_list-of-goods/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -26,70 +27,133 @@ type ReorderOptions = {
...
+        return 0;
+    }
+  });
+
+  if (isReversed) {
+    visibleGoods.reverse();
+  }
+
   // eslint-disable-next-line no-console
   console.log(sortType, isReversed);
```
Comment (oleh-yablunovskyi) [inline src/App.tsx:54]: No need to left this console.log in code

**Review by oleh-yablunovskyi — APPROVED** [review]
Good job!

File: src/App.tsx [hunk 2]
```diff
@@ -26,70 +27,133 @@ type ReorderOptions = {
...
+      case SortType.ALPHABET:
+        return fistGood.localeCompare(secondGood);
+
+      case SortType.LENGTH:
+        return Number(fistGood.length) - Number(secondGood.length);
+
+      default:
+        return 0;
+    }
+  });
```
Comment (ivan-kadykalo) [inline src/App.tsx:47]: Don't start sort if there is no sort type + use switch case before sort, not iside
```suggestion
  switch (sortType) {
    case SortType.ALPHABET:
      visibleGoods.sort((fistGood, secondGood) => (
        fistGood.localeCompare(secondGood);
      )

      case ....
```

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]
You are almost there 💪

**Pull request closed without merging**